brian_cheung/week_two_challenges/kanye_records.md

```diff
@@ -67,9 +63,9 @@ end
 ```ruby
 kanyeRecords.each do |i|
   if i[:title].length >= 15
-    puts "'#{:title}' That's a long title"
+    puts "'#{i[:title]}' That's a long title"
   else
-    puts "'#{:title}' That's a short title"
+    puts "'#{i[:title]}' That's a short title"
   end
 end
 ```
```

Comment:
Again with the naming of `i` here - you should aim to give your variables names that clarify what they actually represent.